api/requirements.txt

```diff
@@ -1,42 +1,42 @@
-Flask==1.0.2
-Flask-Admin==1.5.3
-Flask-Bcrypt==0.7.1
-Flask-Failsafe==0.2
-Flask-Migrate==2.3.0
-Flask-RESTful==0.3.7
-Flask-Redis==0.3.0
-flask-restplus==0.13.0
-Flask-SQLAlchemy==2.3.2
-Flask-Script==2.0.6
+Flask
```

Comment:
Should we completely unpin the package versions? With the upgrade I think the case can be made to take the latest versions now can also see how that would be quite tedious 🙈

Reply:
They are completely unpinned now except for Werkzeug which has a version incompatibility with the latest Flask-restful, unless I didn't understand correctly what you mean?